GraphQL.md

```diff
@@ -19,47 +19,70 @@ POST https://yourdomain.com/graphql
 The structure for our GraphQL payloads is generated and, therefore, uniform across all data types. To understand our format, it is strongly recommended that you first have a [basic understanding of GraphQL](http://graphql.org/learn/). The general structure is as follows:
 
 ```
-rootObjectTypeName([op: FETCH|ADD|REPLACE|DELETE, id: ID-VALUE, data: DATA-OBJ]) {
-  exposedFields
+rootObjectTypeName([op: UPSERT|REPLACE|REMOVE|DELETE, ids: [ID-VALUES], data: DATA-OBJ]) {
+  exposedFields,
+  objectName([op: UPSERT|REPLACE|REMOVE|DELETE, ids: [ID-VALUES], data: DATA-OBJ])
 }
 ```
 
 where `rootObjectTypeName` is the name of an [Elide rootable](http://elide.io/pages/guide/01-start.html) object. Any object then can take any of 3 optional arguments.
 
-  * `op` (abbreviated for `operation`) is one of `FETCH`, `REPLACE`, `ADD`, or `DELETE`. The default value is `FETCH` when unspecified.
-  * `id` is the identifier for an object. If this value is unspecified the system assumes to be working on a collection of these objects. Otherwise, it is working on the specified id.
-  * `data` is the data input object that is used as the argument to the specified `op` argument. This argument is unused in the case of `FETCH` and `DELETE`.
+  * `op` (abbreviated for `operation`) is one of `UPSERT`, `REPLACE`, `REMOVE` or `DELETE`. The default value is `UPSERT` when unspecified.
+
+     * `UPSERT` - inserts new or edits existing members of a relationship. It is the primary mechanism by which any updates will be made to the model.
+     * `REPLACE` - can be thought as a combination of `UPSERT` and `REMOVE`, wherein, any objects not specified in the data argument (that are visible to the user) will be removed.
+     * `REMOVE` - only disassociates a relationship. If the underlying store automatically deletes orphans, it will still be deleted.
+     * `DELETE` - both disassociates the relationship and it deletes the entity from the persistence store.
+
+  
+  * `ids` are used as the ientifier for object(s). This field can hold a singleton value (for a single object) or a list of values to specify multiple ids. If this value is unspecified the system assumes to be working on a collection of these objects. Otherwise, it is working on the specified id.
```

Comment:
Similar to how you call out that 'data' is unsupported for some operations, you should do the same for 'ids' here.